tests/iteminfo_test.cpp

```diff
@@ -638,14 +638,14 @@ TEST_CASE( "armor coverage, warmth, and encumbrance", "[iteminfo][armor][coverag
 
         // Coverage and warmth are displayed together on a single line
         std::vector<iteminfo_parts> cov_warm = { iteminfo_parts::ARMOR_COVERAGE, iteminfo_parts::ARMOR_WARMTH };
-        REQUIRE( longshirt.get_coverage() == 90 );
+        REQUIRE( longshirt.get_avg_coverage() == 90 );
         REQUIRE( longshirt.get_warmth() == 5 );
         CHECK( item_info_str( longshirt, cov_warm )
                ==
                "--\n"
-               "Coverage: <color_c_yellow>90</color>%  Warmth: <color_c_yellow>5</color>\n" );
+               "Average Coverage: <color_c_yellow>90</color>%  Warmth: <color_c_yellow>5</color>\n" );
 
-        REQUIRE( longshirt.get_encumber( g->u ) == 3 );
+        REQUIRE( longshirt.get_encumber( g->u, bodypart_id( "num_bp" ) ) == 3 );
```

Comment:
It would be good to see tests in `iteminfo_test` covering the new `iteminfo` output you've written code for.

Reply:
I agree, I am having a bit of trouble writing tests that pass correctly as I don't quite understand the test framework well enough, but it is something that would be nice to have for sure.